cmd/exchange_template/exchange_template_test.go

```diff
@@ -65,18 +66,14 @@ func TestNewExchangeAndSaveConfig(t *testing.T) {
 			WS:   true,
 		},
 	)
```

Comment:
🐒 🤷‍♂️  Is testExchangeDir replaceable with `t.TempDir()`, and remove the `t.Cleanup` ?